src/app/components/Form/Form.tsx

```diff
@@ -3,27 +3,39 @@ import classes from './Form.module.css';
 
 export default function InputForm(): JSX.Element {
   return (
-    <form action="">
-      <label className={classes.labelTitle}>
-        Name des Gegenstandes
-        <br />
-        <input type="text" autoComplete="off" />
-      </label>
-      <label>
-        Beschreibung des Gegenstandes
-        <br />
-        <textarea rows={5}></textarea>
-      </label>
-      <label>
-        Gewicht
-        <br />
-        <br />
-        <input type="number" defaultValue={0.5} min={0.5} max={25} step={0.5} />
-        <br />
-      </label>
+    <form className={classes.input_form} action="">
       <br />
-      <br />
-      <input className={classes.inputButton} type="submit" value="Submit" />
+      <input
+        className={classes.input_field}
+        type="text"
+        autoComplete="off"
+        placeholder="Name des Gegenstandes"
+      />
+      <div className={classes.spacing_div} />
+      <textarea
+        className={classes.input_field}
+        rows={5}
+        placeholder="Beschreibung des Gegenstandes"
+      ></textarea>
+      <div className={classes.spacing_div} />
+      <div className={classes.spacing_div} />
+      <label className={classes.lable_weight}>
+        Gewicht in Kilogramm <br />
+        (0,2kg Schritte!)
```

Comment:
I suggest to use a span element here with display:block; this way, you can style the comment if you want and you don't rely on <br/>
```suggestion
        <span>(0,2kg Schritte!)</span>
```